test/ui-test/src/screens/StatusMainScreen.py

```diff
@@ -110,7 +110,7 @@ def open_settings(self):
         time.sleep(0.5)
 
     def open_start_chat_view(self):
-        click_obj_by_name(MainScreenComponents.START_CHAT_BTN.value)
+        Button(MainScreenComponents.START_CHAT_BTN.value).click(1, 1)
```

Comment:
Question: What does it mean this 1, 1 params? It's not intuitive imo. If it is a default value to click on an object, maybe it should be just something done in the base class.